addons/account/models/account_move.py

```diff
@@ -1983,7 +1983,7 @@ def write(self, vals):
                 raise UserError(_('You cannot overwrite the values ensuring the inalterability of the accounting.'))
             if (move.posted_before and 'journal_id' in vals and move.journal_id.id != vals['journal_id']):
                 raise UserError(_('You cannot edit the journal of an account move if it has been posted once.'))
-            if (move.name and move.name != '/' and 'journal_id' in vals and move.journal_id.id != vals['journal_id']):
+            if (move.name and move.name != '/' and not move.name.endswith('0001') and 'journal_id' in vals and move.journal_id.id != vals['journal_id']):
```

Comment:
This is too weird @william-andre

Reply:
Yeah, you probably want to use the field `sequence_number` instead
And you can also just clear the sequence anyway.

> You cannot edit the journal of an account move if it already has a sequence number assigned.

Oh then, I should not have a sequence number. Let's try to remove the sequence. Oh it works!

Reply:
IMO, removing the sequence is quite annoying for the user if he doesn't want to set it manually, I would prefer use the field sequence_number as you said.